hms-app/src/actions/GetBackendData.tsx

```diff
@@ -0,0 +1,52 @@
+const api_id = '6cyvp5ukhc'
+
+export const getListOfHackathons = (endpoint: any) => {
```

Comment:
the name here implies that that `endpoint` will always be `hackathons` so you likely don't need to pass in a value.  You should also specify a return type here :thinking: